Integrations/integration-UrlScan.yml

```diff
@@ -0,0 +1,378 @@
+description: Urlscan.io reputation
+detaileddescription: |-
+  This integration checks domain information from the Urlscan.io Database.
+  No authentication is needed.
+  If you want to submit a scan you need an API key.
+  The API key provided by contacting the Urlscan.io directly  by email.
+configuration:
+- display: URL for Urlscan API
+  name: url
+  defaultvalue: https://Urlscan.io/api/v1/
```

Comment:
no capital U in url
in general - have the entire code say urlscan.io when you're mentioning the service